proto/cmp/services/activity/v1alpha/activity_types.proto

```diff
@@ -0,0 +1,200 @@
+syntax = "proto3";
+
+// ### Activity Services
+//
+// The Activity services are used for both tickets and excursions
```

Comment:
We don't need this here. 

(This is package level comment and there should be only one in the package, which we put in the search.proto file)